cg/store/database.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, scoped_session, sessionmaker
+
+from cg.exc import CgError
+from cg.store.models import Model
+
+SESSION: Optional[Session] = None
+ENGINE = None
+
+
+def initialise_database(db_uri: str):
+    global SESSION, ENGINE
+    ENGINE = create_engine(db_uri, pool_pre_ping=True)
+    session_factory = sessionmaker(ENGINE)
+    SESSION = scoped_session(session_factory)
+
+
+def get_session() -> Session:
+    if not SESSION:
+        raise CgError("Database not initialised")
+    return SESSION()
```

Comment:
```suggestion
    return SESSION
```

Reply:
I just see this is the same as in https://github.com/Clinical-Genomics/housekeeper/pull/151 

is the `()` wanted? It is a bit confusing maybe a docstring is warranted here

Reply:
Yes, `SESSION` is a factory - see [session docs](https://docs.sqlalchemy.org/en/20/orm/contextual.html#contextual-thread-local-sessions)